src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -39,17 +39,22 @@ const OTHER_COLOR = "#8B00C9";
 export default class MutationMapper extends React.Component<IMutationMapperProps, {}>
 {
     @observable protected is3dPanelOpen = false;
+    @observable lollipopPlotGeneX:number = 0;
+    @observable geneWidth:number = 665;
```

Comment:
Is there a specific reason to make this an observable, instead of a default property?